src/init.js

```diff
@@ -22,7 +22,7 @@ export default function init(data) {
     layout.call(this);
 
     //Initialize charts.
-    multiply(this, this.data.raw, 'measure_unit');
+    multiply(this, this.data.raw, 'measure_unit', this.config.allMeasures);
 
```

Comment:
So that cells will draw even when there is no data?

Reply:
So the multiples draw in alphabetical order.  Otherwise they draw as they appear in the data.